Functions/xASL_adm_LoadParms.m

```diff
@@ -1,7 +1,7 @@
-function [Parms, x, Oldx] = xASL_adm_LoadParms(ParmsPath, x, bVerbose)
-%xASL_adm_LoadParmsMat Loads parameters from .mat or .json sidecar
+function [Parms, x] = xASL_adm_LoadParms(ParmsPath, x, bVerbose)
```

Comment:
Are you sure we don't use the OldX anymore (I assume you are but just to be sure :)

Reply:
Yes. Checked the code and searched through all uses of it everywhere.